### PR TITLE
Added option to run http-server

### PR DESCRIPTION
**Pull request by jimmynotjim**
Added http-server instructions to the readme, a small npm script, and a dependency for that script to make it easier to run a fresh project or test the generator and Capital Framework itself.

## Additions

- NPM script to start the server and watch tasks
- Parallelshell package for the NPM script
- Instructions on how to install and run the server.

## Testing

- Sym-link the generator by running `npm link` in the repo root
- In an empty test directory run `yo cf`, follow the prompts
- In a new terminal window at the test directory root, run `npm start`
- A server will start using port `7000` with a browsersync session on port `3000`

## Review

- @contolini 
- @anselmbradford 
- @Scotchester 

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: app/templates/grunt/_README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ grunt watch:cssjs
 After running `./setup.sh` or compiling with Grunt,
 you can view the site in a browser by opening `/dist/index.html`.
 Alternatively, you may want to use a local server with something like
-`python -m SimpleHTTPServer`.
+`http-server`. To do so, run `npm install http-server -g` and `npm start`.
```
Comment (Scotchester) [inline app/templates/grunt/_README.md]: Is it worth mentioning that this script will also turn on `grunt watch`?

File: app/templates/grunt/_package.json [hunk 2]
```diff
@@ -39,6 +42,7 @@
     "grunt-string-replace": "1.2.0",
     "grunt-topdoc": "0.3.0",
     "jit-grunt": "0.9.1",
+    "parallelshell": "2.0.0",
```
Comment (Scotchester) [inline app/templates/grunt/_package.json]: Is `parallelshell` the kind of thing we should encourage people to install globally?

Comment (jimmynotjim) [inline app/templates/grunt/_package.json]: @Scotchester I guess we could, but per-project seems safer, doesn't it?

File: app/templates/grunt/_README.md (hunk 1 above)
Comment (jimmynotjim) [inline app/templates/grunt/_README.md]: Good point, adding it to both the Grunt and Gulp instructions

Comment (jimmynotjim) [inline app/templates/grunt/_README.md]: Good point, adding it to both the Grunt and Gulp instructions

File: app/templates/grunt/_package.json (hunk 2 above)
Comment (jimmynotjim) [inline app/templates/grunt/_package.json]: Updated to install `http-server` as a local dep rather than require another globally installed dep.

**Pull request merged**